src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2226,6 +2226,15 @@ namespace winrt::TerminalApp::implementation
         {
             _newTabButton.Flyout().Hide();
         }
+
+        for (auto tab : _tabs)
```

Comment:
```suggestion
        for (const auto& tab : _tabs)
```

may save a few cycles doing AddRef/Release on each tab